<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenDeposit</class>
 <widget class="QDialog" name="OpenDeposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>190</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:xx-large; font-weight:700;&quot;&gt;Deposit&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Меню</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="depositAmountInput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите первоначальный взнос</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок вклада</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="depositTermInput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="depositCheck">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Непополняемый вклад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="depositOpenButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть вклад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
